lib/cfndsl/Types.rb

```diff
@@ -45,12 +39,15 @@ class AWSType < JSONable
     
     # Go through and declare all of the types first
     aws_types["Types"].each_key do |typename|
-      if( ! AWSTypes.const_defined? typename ) then
-        klass = AWSTypes.const_set( typename, Class.new(AWSType ) )
-        classes[typename] = klass
-      else
-        classes[typename] = AWSTypes.const_get(typename)
-      end 
+      next if %w(String JSON Integer).include?(typename)
+
+      if AWSTypes.const_defined? typename then
+        puts "duplicate type #{typename}"
```

Comment:
@johnf Do we really want to be outputting this? If there are duplicate types we should either remove them or not alter the output so there's a number of `duplicate type` outputs whenever people run the command..